simplified-main/src/main/java/org/nypl/simplified/main/MainApplication.kt

```diff
@@ -69,6 +73,30 @@ class MainApplication : MultiDexApplication() {
     }
   }
 
+  /**
+   * StrictMode is a developer tool which detects things you might be doing by accident and
+   * brings them to your attention so you can fix them.
+   *
+   * StrictMode is most commonly used to catch accidental disk or network access on the
+   * application's main thread, where UI operations are received and animations take place.
+   */
+
+  private fun configureStringMode() {
```

Comment:
`StringMode` -> `StrictMode` :laughing:

Reply:
Wow, that's embarrassing! I should not be doing this so late at night 😂